rootfs/deis/production.py

```diff
@@ -1,6 +1,4 @@
-import os
```

Comment:
I think `production_settings.py` is more fitting, or does that clash with `--settings deis.production`?